metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/usage.py

```diff
@@ -66,7 +66,7 @@ class OperationalDataMeta:
 def bigquery_audit_metadata_query_template(
     dataset: str,
     use_date_sharded_tables: bool,
-    table_allow_filter: str = None,
+    table_allow_filter: Optional[str] = None,
```

Comment:
was this change necessary?

Reply:
This field should be Optional and mypy (linter) was complaining about it.